useragent/mac/dnsfilter_ua_debugger_mac.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+echo "======================================================" >> $ResultLog
+echo "DNSFilter Debugger for Mac"
+echo "The purpose of this tool is to assist support engineers with common requests."
+#CREATED BY RICK COHEN DEC 2022
+echo "======================================================" >> $ResultLog
+# Create the "DNSFilter Logs" directory if it doesn't exist
+if [ ! -d "$HOME/Downloads/DNSFilterLogs" ]; then
+  mkdir $HOME/Downloads/DNSFilterLogs
+fi
+
+# Store all results to this file
+LogFolder=$HOME/Downloads/DNSFilterLogs
+ResultLog=$HOME/Downloads/DNSFilterLogs/DNSFilterDebug.txt
```

Comment:
Variable declaration should be at the beginning of the file, otherwise the first few lines are outputting to a non-existent variable